superset-frontend/src/pages/SavedQueryList/SavedQueryList.test.jsx

```diff
@@ -302,9 +302,13 @@ describe('RTL', () => {
     userEvent.click(bulkSelectButton);
 
     // Grab and click the "toggle all" checkbox to expose export button
-    const selectAllCheckbox = screen.getByRole('checkbox', {
-      name: /toggle all rows selected/i,
-    });
+    const selectAllCheckbox = screen
+      .getAllByRole('checkbox', {
+        name: '',
+      })
+      .filter(
+        checkbox => checkbox.getAttribute('name') === 'header-toggle-all',
+      )[0];
```

Comment:
Was there an issue with the previous code? If there is no reason for the changes, selecting functionality based on class names is not recommended.

Reply:
Yes, for some reasons, the `selectAllCheckbox` is not query-able so I had to rely on attribute name as last resort.